src/Module/Api/GNUSocial/GNUSocial/Config.php

```diff
@@ -47,7 +48,7 @@ protected function rawContent(array $request = [])
 				'broughtby'    => '',
 				'broughtbyurl' => '',
 				'timezone'     => DI::config()->get('system', 'default_timezone'),
-				'closed'       => (DI::config()->get('config', 'register_policy') == Register::CLOSED),
+				'closed'       => (Security::getRegisterPolicy() == Register::CLOSED),
```

Comment:
After changing the return type of `getRegisterPolicy`
```suggestion
				'closed'       => Security::getRegisterPolicy() === Register::CLOSED,
```